### PR TITLE
switching from cmalign to cmsearch

### PR DESCRIPTION
**Pull request by crosenth**
Per Issue https://github.com/nhoffman/dada2-nf/issues/13

### CODE REVIEW

**Comment by tholland** [comment]
The only thing I would note is prior pipelines also used the `-E 0.01` flag with `cmsearch`.

**Comment by crosenth** [comment]
@tholland - What was the -E flag used for in prior pipelines?

**Comment by tholland** [comment]
In our case I recall it produced better results to include the stricter threshold for matches, but I'd have to find the validation for it again to see how much of a difference it makes.

File: nextflow.config [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // these values supersede params in the profile and are themselves
 // superseded by contents of -params-file
 params {
-    container = 'quay.io/nhoffman/dada2-nf:v1.12-9-g39fa45b'
+    container = 'dada2-nf:v1.12-153-g4a48721-dirty'
```
Comment (crosenth) [inline nextflow.config]: @nhoffman -  We'll need to update your image on quay.  See docker/install_infernal_and_easel.sh (above)

**Pull request merged**